lib/stytch/passwords.rb

```diff
@@ -1,228 +1,674 @@
+    #   The type of this field is `Boolean`.
+    # score::
+    #   The score of the password determined by [zxcvbn](https://github.com/dropbox/zxcvbn). Values will be between 1 and 4, a 3 or greater is required to pass validation.
+    #   The type of this field is `Integer`.
+    # breached_password::
+    #   Returns `true` if the password has been breached. Powered by [HaveIBeenPwned](https://haveibeenpwned.com/).
+    #   The type of this field is `Boolean`.
+    # strength_policy::
+    #   The strength policy type enforced, either `zxcvbn` or `luds`.
+    #   The type of this field is `String`.
+    # breach_detection_on_create::
+    #   Will return `true` if breach detection will be evaluated. By default this option is enabled. This option can be disabled by contacting [support@stytch.com](mailto:support@stytch.com?subject=Password%20strength%20configuration). If this value is `false` then `breached_password` will always be `false` as well.
+    #   The type of this field is `Boolean`.
+    # status_code::
+    #   The HTTP status code of the response. Stytch follows standard HTTP response status code patterns, e.g. 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX are server errors.
+    #   The type of this field is `Integer`.
+    # feedback::
+    #   Feedback for how to improve the password's strength [HaveIBeenPwned](https://haveibeenpwned.com/).
+    #   The type of this field is nilable `Feedback`.
     def strength_check(
       password:,
       email: nil
     )
       request = {
         password: password
       }
-
       request[:email] = email unless email.nil?
 
-      post_request("#{PATH}/strength_check", request)
+      post_request('/v1/passwords/strength_check', request)
     end
 
+    # Adds an existing password to a User's email that doesn't have a password yet. We support migrating users from passwords stored with `bcrypt`, `scrypt`, `argon2`, `MD-5`, `SHA-1`, or `PBKDF2`. This endpoint has a rate limit of 100 requests per second.
+    #
+    # == Parameters:
+    # email::
+    #   The email address of the end user.
+    #   The type of this field is `String`.
+    # hash::
+    #   The password hash. For a Scrypt or PBKDF2 hash, the hash needs to be a base64 encoded string.
+    #   The type of this field is `String`.
+    # hash_type::
+    #   The password hash used. Currently `bcrypt`, `scrypt`, `argon_2i`, `argon_2id`, `md_5`, `sha_1`, and `pbkdf_2` are supported.
+    #   The type of this field is `MigrateRequestHashType`.
+    # md_5_config::
+    #   Optional parameters for MD-5 hash types.
+    #   The type of this field is nilable `MD5Config`.
```

Comment:
same here - does the caller know what an MD5Config is? Should this just be an `object`?

Reply:
Compromised. It's now like `+MD5Config+ (+object+)`